cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1763,7 +1819,7 @@ public String getMetadataVersion() {
      * @return  Indicates whether this is a KRaft cluster or not
      */
     public boolean usesKRaft() {
-        return useKRaft;
+        return kafkaMetadataConfigState.isKRaft();
```

Comment:
Is this method still needed? You now store the `kafkaMetadataStateManager` in KafkaReconciler, so it probably does not need to be exposed from here?

Reply:
Yeah I can avoid that method.